src/main/java/com/umc/networkingService/domain/album/entity/AlbumComment.java

```diff
@@ -0,0 +1,42 @@
+package com.umc.networkingService.domain.album.entity;
+
+import com.umc.networkingService.domain.member.entity.Member;
+import com.umc.networkingService.global.common.BaseEntity;
+import jakarta.persistence.*;
+import lombok.AccessLevel;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import org.hibernate.annotations.UuidGenerator;
+
+import java.util.UUID;
+
+@Entity
+@Getter
+@NoArgsConstructor(access = AccessLevel.PROTECTED)
+public class AlbumComment extends BaseEntity {
+
+    @Id
+    @UuidGenerator
+    @Column(name = "album_comment_id")
+    private UUID id;
+
+    @ManyToOne(fetch = FetchType.LAZY)
+    @JoinColumn(nullable = false,name = "member_id")
+    private Member writer;
+
+    @ManyToOne(fetch = FetchType.LAZY)
+    @JoinColumn(nullable = false,name = "album_id")
+    private Album album;
+
+
+    //최상위 댓글인 경우 null
+    @ManyToOne(fetch = FetchType.LAZY)
+    @JoinColumn(name="parent_comment_id")
+    private AlbumHeart parentComment;
```

Comment:
```suggestion
    private AlbumComment parentComment;
```
객체를 잘 못 지정해줘서 변경해야 할 거 같아요